roles/grafana/tasks/main.yml

```diff
@@ -17,12 +17,29 @@
     name: "{{ __grafana_packages + __grafana_packages_extra }}"
     state: present
 
-- name: Template Grafana configuration
+- name: Get package facts now that Grafana is installed
+  package_facts:
+    manager: "auto"
+
+- name: Get Grafana version number from installed packages
+  set_fact:
+    grafana_version: "{{ ansible_facts.packages.grafana[0].version }}"
+
+- name: Template Grafana v7 configuration
```

Comment:
(picking this up while Andreas is away to get it merged)

There was - the way Andreas has the code now its using before/after v9 logic (so the "v7" config is also used on v8, v5, ...).  Which is how it was previous to this change.  I'm going to assert that this continues to be a safe thing to do since there was no known issues before v9.